language-management/src/js/views/modalView.js

```diff
@@ -1,9 +1,11 @@
 import Event from "../events/event";
+import { MESSAGE } from "../constants/message";
 
 class ModalView {
```

Comment:
Should choose a name that describes what you want to do. i.e. `AddEditAddressModal`